tsl/test/expected/compression_qualpushdown.out

```diff
@@ -195,6 +195,7 @@ SELECT compress_chunk(i) from show_chunks('pushdown_relabel') i;
  _timescaledb_internal._hyper_5_6_chunk
 (1 row)
 
+ANALYZE pushdown_relabel;
```

Comment:
you seem to be adding all these `analyze` staments because the cost estimates for seqscan have changed:

<details>
<summary>no analyze current patch</summary>

```
                                                                       QUERY PLAN                                                                       
--------------------------------------------------------------------------------------------------------------------------------------------------------
 Custom Scan (DecompressChunk) on _hyper_5_6_chunk  (cost=1.59..3.18 rows=2000 width=27)
   ->  Index Scan using compress_hyper_6_7_chunk__compressed_hypertable_6_dev_vc_dev_c_ on compress_hyper_6_7_chunk  (cost=0.15..3.18 rows=2 width=134)
         Index Cond: ((dev_vc)::text = 'varchar'::text)
```

</details>
<details>
<summary>after analyze - current patch</summary>

```
                                       QUERY PLAN                                        
-----------------------------------------------------------------------------------------
 Custom Scan (DecompressChunk) on _hyper_5_6_chunk  (cost=1.01..1.01 rows=1000 width=27)
   ->  Seq Scan on compress_hyper_6_7_chunk  (cost=0.00..1.01 rows=1 width=71)
         Filter: ((dev_vc)::text = 'varchar'::text)
(3 rows)
```

</details>
<details>
<summary>no analyze ; forced seqscan; current patch</summary>

```
                                        QUERY PLAN                                        
------------------------------------------------------------------------------------------
 Custom Scan (DecompressChunk) on _hyper_5_6_chunk  (cost=8.06..16.12 rows=2000 width=27)
   ->  Seq Scan on compress_hyper_6_7_chunk  (cost=0.00..16.12 rows=2 width=134)
          Filter: ((dev_vc)::text = 'varchar'::text)
```

</details>
<details>
<summary>without patch</summary>

```
 Custom Scan (DecompressChunk) on _hyper_5_6_chunk  (cost=1.01..1.01 rows=1000 width=27)
   ->  Seq Scan on compress_hyper_6_7_chunk  (cost=0.00..1.01 rows=1 width=134)
          Filter: ((dev_vc)::text = 'varchar'::text)
```

</details>

it seems like the `width` is also more precise after `analyze` not sure if that matters

without the analyze the cost of the seqscan gone up from `0.00..1.01`  to `0.00..16.12` - what's the reason for that?

Reply:
Running compress_chunk without patch would update the pg_class statistics because we were doing reindex_relation. Patch removes it this so in order to update the statistics, we run analyze to update them manually thus not changing any test plans.

Not completely sure why without updating the statistics PG seems to think there are two rows in the compressed chunk since there is clearly only one.

Reply:
If you look at the
* `after analyze - current patch`
* `without patch`

the system was able to correctly compute the right rowcount incrementally before this patch

that incremental maintanence is broken by this patch...